fs.go

```diff
@@ -770,11 +805,22 @@ func (h *fsHandler) handleRequest(ctx *RequestCtx) {
 		return
 	}
 
-	r, err := ff.NewReader()
-	if err != nil {
-		ctx.Logger().Printf("cannot obtain file reader for path=%q: %s", path, err)
-		ctx.Error("Internal Server Error", StatusInternalServerError)
-		return
+	var r io.Reader
+	if loaded {
+		r, err = ff.NewBufferReader()
```

Comment:
When an `Accept-Ranges` header is send, code below will unconditionally casts `r` to an `io.Closer` and `byteRangeUpdater` both of which aren't implemented by `bytes.Buffer` and will cause a panic.

Reply:
@themester you didn't do anything about this?

Reply:
Yes. I changed it here https://github.com/erikdubbelboer/fasthttp/pull/32/commits/ad26055ae8eac5d6f284794b594328e74b5974c8